src/status_im/ui/screens/db.cljs

```diff
@@ -78,7 +78,9 @@
 ;;object?
 (spec/def ::webview-bridge (spec/nilable any?))
 (spec/def ::status-module-initialized? (spec/nilable boolean?))
-(spec/def ::status-node-started? (spec/nilable boolean?))
+(spec/def ::status-node-state (spec/nilable #{:stopped :starting :started :stopping}))
+(spec/def ::dispatch-after-start (spec/nilable vector?))
```

Comment:
I would simply have `:node/restart?` and that would be nilable boolean
and `:node/address` to store the node address if it can't be retrieve somewhere else